local-cluster/tests/local_cluster.rs

```diff
@@ -4273,49 +4295,6 @@ fn test_leader_failure_4() {
     );
 }
 
-#[test]
-#[serial]
-fn test_ledger_cleanup_service() {
```

Comment:
why is this removed? it was failing?

Reply:
Yeah, see comment here: https://github.com/anza-xyz/agave/pull/696#issuecomment-2047858495